plugin/storage/es/spanstore/reader_test.go

```diff
@@ -38,6 +38,45 @@ import (
 	"github.com/uber/jaeger/storage/spanstore"
 )
 
+var exampleESSpan = []byte(`{      "traceID": "1",
```

Comment:
can this be formatted a bit better? there's too much indentation, start the json blob on a new line